ci-operator/jobs/openshift/knative-serving/openshift-knative-serving-release-v0.11.1-postsubmits.yaml

```diff
@@ -1,112 +0,0 @@
-postsubmits:
```

Comment:
Same comment as before. Should we keep this because we're keeping serverless release-1.4?